Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```diff
@@ -5283,20 +5269,20 @@ void WebGLRenderingContextBase::texImageArrayBufferViewHelper(TexImageFunctionID
     }
 #if USE(ANGLE)
     if (functionID == TexImageFunctionID::TexImage3D) {
-        texImage3DBase(target, level, internalformat, width, height, depth, border, format, type, byteLength, data);
+        texImage3DBase(target, level, internalformat, width, height, depth, border, format, type, byteLength, data, &bytesRequired);
```

Comment:
the bytesRequired is not a out parameter, so there shouldn't be a ptr passed here.

Reply:
I was using pointer as "optional" parameter.